test/functional/p2p_segwit.py

```diff
@@ -220,7 +218,77 @@ def update_witness_block_with_transactions(self, block, tx_list, nonce=0):
         block.vtx.extend(tx_list)
         add_witness_commitment(block, nonce)
         block.solve()
-        return
```

Comment:
In commit 2af4e39 [tests] p2p_segwit: re-order function definitions.

Unrelated non-moveonly removal